AltServer/Base.lproj/Main.storyboard

```diff
@@ -132,7 +134,7 @@
                                         <menuItem title="Check for Updates..." id="Tnq-gD-Eic" userLabel="Check for Updates">
                                             <modifierMask key="keyEquivalentModifierMask"/>
                                             <connections>
-                                                <action selector="checkForUpdates:" target="Arf-IC-5eb" id="7JG-du-nr4"/>
+                                                <action selector="checkForUpdates:" target="9QV-ha-p2H" id="9fB-b9-def"/>
```

Comment:
Ah, so was the check for updates already there, and you're just reworking it somehow?